modules/ui/panels/history.js

```diff
@@ -40,7 +40,7 @@ export function uiPanelHistory(context) {
                 .attr('class', 'user-osm-link')
                 .attr('href', osm.userURL(userName))
                 .attr('target', '_blank')
-                .text('OSM');
+                .text('OSM Profile');
```

Comment:
Also call `t.append` here. The argument to `t.append` should refer to an entry in English that you’ll need to add to in data/core.yaml. Once this PR lands, translators will be able to provide translations using Transifex, so there’s no need to handle the translations yourself.